ios/core/Sources/Types/Core/ConstantsController.swift

```diff
@@ -0,0 +1,31 @@
+import JavaScriptCore
+
+public class ConstantsController {
+    var constantsController: JSValue?
+
+    public func getConstants<T>(key: Any, namespace: String, fallback: Any? = nil) -> T? {
+        if let fallbackValue = fallback {
+            let value = self.constantsController?.invokeMethod("getConstants", withArguments: [key, namespace, fallbackValue])
+            return value?.toString() as? T
+        } else {
+            let value = self.constantsController?.invokeMethod("getConstants", withArguments: [key, namespace])
+            return value?.toString() as? T
+        }
+    }
+
+    public func addConstants(data: Any, namespace: String) -> Void {
```

Comment:
We should add docstrings to all public functions

Xcode can scaffold the docstrings for you if you right click a function and do `Add Documentation`